script.js

```diff
@@ -1,406 +1,292 @@
-      '1 large onion, quartered',
-      '2 carrots, sliced',
-      '1 leek, cleaned and sliced',
-      '2 celery stalks, sliced',
-      '2.5 pounds organic beef stew meat, cubed',
-      '2 tablespoons tomato paste',
-      '5 cloves garlic',
-      '2 bay leaves',
-      '3 sprigs thyme',
-      '3 sprigs Italian parsley',
-      '1/2 teaspoon black peppercorns'
-    ],
-    source: 'Food52',
-    totalTime: 60,
-    url: 'https://food52.com/recipes/3712-meat-stock',
-    image: './recipe-images/meat.jpg'
-  },
-  {
-    name: 'Homemade Meat Broth',
-    cuisineType: 'american',
-    ingredients: [
-      '1 teaspoon salt',
-      '1 carrot, peeled',
-      '1 medium onion, peeled',
-      '2 or 3 celery stalks',
-      '¼ red or yellow bell pepper, stripped of all its seeds',
-      '1 small boiling potato, peeled',
-      '1 ripe, fresh tomato, or 1 canned Italian plum tomato, drained of juice',
-      '5 pounds assorted pieces of meat and bones (see meat suggestions above), of which no less than 1½ pounds is all meat'
-    ],
-    source: 'Cookstr',
-    totalTime: 60,
-    url: 'http://www.cookstr.com/recipes/il-brodo-homemade-meat-broth',
-    image: './recipe-images/meat.jpg'
-  },
-  {
-    name: 'Spice-Rubbed Grilled Flap Meat (Sirloin Tip) Recipe',
-    cuisineType: 'south-american',
-    ingredients: [
-      '1 tablespoon whole black peppercorns, toasted',
-      '1 teaspoon coriander seed, toasted',
-      '1 teaspoon fennel seed, toasted',
-      '1 teaspoon cumin pods, toasted',
-      '1 teaspoon red pepper flakes',
-      '1/2 teaspoon dried oregano',
-      '2 medium cloves garlic, minced (about 2 teaspoons)',
-      '2 tablespoons vegetable or canola oil',
-      '1 whole flap meat steak, 2 to 2 1/2 pounds',
-      'Kosher salt'
-    ],
-    source: 'Serious Eats',
-    totalTime: 240,
-    url: 'http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html',
-    image: './recipe-images/grilled.jpg'
+//Function that loads the content of the objects into the HTML
+const loadDestinations = (destinations) => {
+    // Cleans the div before each load
+    destinationDiv.innerHTML = ""
+    destinations.forEach((dest) => {
+    destinationDiv.innerHTML += `
+    <div class="container">
+      <div class="text-above"><h4>${dest.name}<br><span>${dest.country}</span></div>
+      <div class="img-container"><img src="${dest.image}"></div>
+      <ul>. 
+        <li>Language: ${dest.language}</li>
+        <li>Accessibility: ${dest.accessibility}</li>
+        <li>Temperature: ${dest.temperature}ºC</li>
+        <li>Rating: ${dest.rating}/5</li>
+      </ul>
+      <div class="description-text"><p>${dest.description}</p></div>
+      </div>
+    `
+    })
+}
+// Calling the function as soon as the website opens
+loadDestinations(destinations)
+
+//function that shift the value each time button is clicked, to enable toggle
+const tempToggle = () => {
+  lowToHigh = !lowToHigh
+  sortByTemp()
+}
+
```

Comment:
We love this, so simple and sweet! We will shamelessly steal this to our corrections. =)

/Martin & Mai